benchmark-service/src/main/resources/application.yml

```
@@ -31,6 +31,8 @@ spring:
 
 server:
   port: '8082'
+  user: 'admin'
+  password: '$2y$08$yrbEY7O0.WjYnGolNmHJN.ymasX5ETmidei162v87yhygZFoEcLHu'
```

That shouldn't be here.
- if key `server.password` is not present, generated it at startup and printed to log
- if present, use it as it is here
```suggestion
  # password: 
```